package.json

```diff
@@ -5,7 +5,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "webpack",
+    "start": "webpack --config webpack.config.js",
```

Comment:
- [ ] Kindly update the scripts and replace them with this so we can start the server and build the project properly 👍 

``` 
  "test": "echo \"Error: no test specified\" && exit 1",
  "start": "webpack serve --open",
  "build": "webpack"
```